apollo-compiler/src/main/antlr/com/apollographql/apollo/compiler/parser/antlr/GraphQL.g4

```diff
@@ -96,7 +96,7 @@ directives
    ;
 
 directive
-   : '@' NAME ':' valueOrVariable | '@' NAME | '@' NAME '(' argument ')'
+   : '@' NAME arguments?
```

Comment:
Not sure what the ':' was up there but I didn't find it in http://spec.graphql.org/draft/#Directives so I'm assuming it's safe to remove that case.